Vic2ToHoI4/Source/Hoi4Outputter/Leaders/OutAdvisor.cpp

```diff
@@ -0,0 +1,30 @@
+#include "OutAdvisor.h"
+
+
+
+void HoI4::outputAdvisor(std::ostream& output, const std::string& tag, const Advisor& theAdvisor)
```

Comment:
turn this into << operator

Reply:
It also needs the tag, which is more items than the operator can handle.